src/app/components/MediaLoader/index.tsx

```diff
@@ -25,17 +27,25 @@ const BumpLoader = () => (
 );
 
 const MediaContainer = ({ playerConfig }: { playerConfig: PlayerConfig }) => {
-  const playerElementRef = useRef(null);
+  const playerElementRef = useRef<HTMLDivElement>(null);
+  // Used to ensure SMP is only loaded once
+  const smpLoadedRef = useRef(false);
```

Comment:
Does the SMP try to load multiple times?

Reply:
I experienced an issue when 'autoplay' was enabled that would cause the player to play twice. Doesn't look like it created multiple elements or anything, just seemed to playback 2 streams when autoplay was enabled.

Its a bit of a workaround this, so I'll take a closer look in subsequent tickets to see if its a problem with re-renders that we need to solve.

Reply:
Ah interesting - thanks for confirming

Reply:
Figured it out. Its because we use `reactStrictMode` in Next.js: https://nextjs.org/docs/pages/api-reference/next-config-js/reactStrictMode

This is a development only thing that causes side-effects like `useEffect` to run twice as it must do some checking of them under-the-hood in dev mode.

Could remove the workaround and maybe note in the ReadMe for the Medialoader that this will happen?

Reply:
Thanks for confirming/ doing this.